lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.spec.ts

```diff
@@ -0,0 +1,108 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { AlfrescoApiService, NotificationService, setupTestBed } from '@alfresco/adf-core';
+import { ProcessServiceCloudTestingModule } from '../../testing/process-service-cloud.testing.module';
+import { TranslateModule } from '@ngx-translate/core';
+import { ContentCloudNodeSelectorService } from 'process-services-cloud';
+import { MatDialog, MatDialogModule } from '@angular/material/dialog';
+import { of, Subject } from 'rxjs';
+
+describe('ContentCloudNodeSelectorService', () => {
+    let service: ContentCloudNodeSelectorService;
+    let apiService: AlfrescoApiService;
+    let notificationService: NotificationService;
+    let getNodeSpy: jasmine.Spy;
+    let dialog: MatDialog;
+    let openDialogSpy: jasmine.Spy;
+    let showWarningSpy: jasmine.Spy;
+
+    const relativePathNodeResponseBody = {
+        entry: {
+            id: 'mock-relative-path-node-id'
+        }
+    };
+
+    const aliasNodeResponseBody = {
+        entry: {
+            id: 'mock-alias-node-id'
+        }
+    };
+
+    setupTestBed({
+        imports: [TranslateModule.forRoot(), ProcessServiceCloudTestingModule, MatDialogModule]
+    });
+
+    beforeEach(() => {
+        service = TestBed.inject(ContentCloudNodeSelectorService);
+        notificationService = TestBed.inject(NotificationService);
+        apiService = TestBed.inject(AlfrescoApiService);
+        dialog = TestBed.inject(MatDialog);
+
+        showWarningSpy = spyOn(notificationService, 'showWarning');
+        openDialogSpy = spyOn(dialog, 'open').and.returnValue({
+            afterOpened: () => of({}),
+            afterClosed: () => of({}),
+            componentInstance: {
+                error: new Subject<any>()
+            }
+        });
+        getNodeSpy = spyOn(apiService.nodesApi, 'getNode');
+    });
+
+    it('should be able to open the content node select panel dialog', () => {
+        service.openUploadFileDialog('nodeId', 'single', true, true);
+        expect(openDialogSpy).toHaveBeenCalled();
+    });
+
+    it('should be able to show an notification if the relative path is invalid/deleted', async () => {
+        getNodeSpy.and.returnValue(Promise.reject('Not exists'));
+        const relativePathNodeEntry = await service.fetchNodeFromRelativePath('mock-alias', { relativePath: 'mock-relativePath' });
+        service.openUploadFileDialog('nodeId', 'single', true, true).subscribe(() => {
+            expect(openDialogSpy).toHaveBeenCalled();
+            expect(service.sourceNodeNotFound).toBe(true);
+            expect(showWarningSpy).toHaveBeenCalledWith('success');
+            expect(relativePathNodeEntry).toBeNull();
+        });
+    });
+
+    it('should not be able to show an notification if the relative path is valid', () => {
```

Comment:
Can we change the title to `should not show a notification if the relative path is valid` ?